src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './utils/variables';
+
```

Comment:
There seems to be a syntax error with the comment on line 2. Comments in SCSS should start with '/*' and end with '*/'. Please correct this to ensure the file is properly commented.